oracle/oracle-explanation/canonical-offerings.rst

```diff
@@ -0,0 +1,57 @@
+Canonical's offerings on Oracle Cloud
+=====================================
+
+Ubuntu images
+-------------
+
+Canonical produces fully optimised Oracle Cloud images that provide a stable base image for running workloads. The two main types of images are:
+
+* **Server images:** General-purpose customised images that include the necessary drivers and kernel modifications to run smoothly in the Oracle Cloud environment. These images launch with the oracle-cloud-agent pre-installed, featuring plugins that collect performance metrics, install OS updates, and perform other instance management tasks.
+
+* **Minimal images:** These are minimal images designed for automated deployment at scale and have a reduced default package set. They are much smaller, boot faster, and require fewer security updates over time due to the fewer installed packages.
+
+Each of these images have variants for AMD64 and ARM64 (Ampere processors). The minimal images for ARM64 are only available from Ubuntu 22.04 onwards. 
+
+The images are built regularly and follow a thorough test framework that helps detect any issues / regressions due to package changes. They are then shared with Oracle privately who publish them on the cloud platform once per month.
+
+With a single image type that is validated to work on both virtual machines and bare metal instances, the images aim to provide a consistent experience for users.
+
+Although only Ubuntu LTS (Long-Term Support) images are publicly available on Oracle Cloud, Canonical also builds Oracle-optimised images for non-LTS releases. This allows for continuous internal testing and ensures that future LTS releases work well on Oracle Cloud.
+
+For instructions on how to find the images, refer to :doc:`../oracle-how-to/find-ubuntu-images`.
+
+
+Optimisations for Oracle Cloud
+------------------------------
+
+Generally the images are built in a qcow2 format and support the following options:
+
+* Boot firmware - boot with BIOS or UEFI (preferred). ARM64 images only support UEFI boot.
+* Launch modes - support for both PARAVIRTUALIZED and NATIVE launch modes.  
+* Network interfaces - support for VFIO and PARAVIRTUALIZED modes.
+* Boot volume -  allows booting through iSCSI or PARAVIRTUALIZED storage.
+
+The customisations include NVMe tunings, iSCSI configurations to allow instances to boot from iSCSI backend when appropriate, logging improvements, iptables and udev configurations specific to Oracle Cloud platform, among others.
+
+
+Kernel optimisations
+--------------------
+
+Ubuntu images available on Oracle Cloud run the linux-oracle kernel. Additional kernel variants are also offered by Canonical, that users can install as appropriate.
```

Comment:
Where are additional kernel variants located / how can they be found?

Reply:
@fabiomirmar or @hammerstefan could you help with this info?

Reply:
The additional kernel variants are available in the APT archives, and they can be installed with `apt install` commands. The variants are:

For x86_64 instances:
- linux-oracle-lts-<release>: Where <release> should be replaced by an LTS Ubuntu version, such as 18.04, 20.04, 22.04 or 24.04. Installing this package will install the latest LTS kernel available, opting out of the rolling kernel strategy, in the sense that the install will have the latest kernel version for the kernel release initially shipped with the Ubuntu LTS release, outside of the enablement stack, which can be seen at https://ubuntu.com/kernel/lifecycle. For example, linux-oracle-lts-22.04 will use the latest 5.15 kernel, instead of using the HWE kernel.
- linux-oracle-edge: Will install the Oracle kernel variant of the -edge kernel, which gives you early access to the next hwe kernel.

For arm64 instances, besides the same variants already mentioned for x86_64 instances, we also have:
- linux-image-oracle-64k: A kernel variant that uses 64k memory pages by default (instead of the usual 4k memory pages), which is optimal for high memory instances, such as Grace Hopper platform, as described at https://discourse.ubuntu.com/t/nvidia-grace-mgx-canonical-support/40942
- linux-headers-oracle-64k-edge: Provides your early access to the next hwe kernel, also configured to use 64k memory pages by default.